packages/docs/src/components/propsTable.tsx

```diff
@@ -49,6 +50,13 @@ const renderPropRow = (prop: IPropertyEntry) => {
             </p>,
         );
     }
+    if (since) {
+        tags.push(
+            <p key="since">
+                <Tag className={Classes.MINIMAL}>Since {since}</Tag>
+            </p>,
+        );
+    }
```

Comment:
Is the vision to populate this automatically—or manually via a `@since x.y.z` in the comment for each relevant prop?